src/cpp/src/gguf_utils/gguf_modeling.cpp

```diff
@@ -141,16 +144,22 @@ std::shared_ptr<ov::Model> create_llama_model(
 } // namespace
 
 std::shared_ptr<ov::Model> create_from_gguf(const std::string& model_path) {
-    auto [config, consts] = load_gguf(model_path);
+    auto start_time = std::chrono::high_resolution_clock::now();
+    std::cout << "Loading and unpacking model from: " << model_path << std::endl;
+    auto [config, consts, qtypes] = load_gguf(model_path);
+    auto load_finish_time = std::chrono::high_resolution_clock::now();
+    std::cout << "Loading and unpacking model done. Time: " << std::chrono::duration_cast<std::chrono::milliseconds>(load_finish_time - start_time).count() << "ms" << std::endl;
 
     std::shared_ptr<ov::Model> model;
 
     const std::string model_arch = std::get<std::string>(config.at("architecture"));
     if (!model_arch.compare("llama") || !model_arch.compare("qwen2")) {
-        model = create_llama_model(config, consts);
+        model = create_llama_model(config, consts, qtypes);
```

Comment:
Does qwen2 re-use all create_llama_model() function? At least reorder_interleaved_format() is not need for Qwen architecture.

Reply:
I see a condition `std::get<std::string>(configs.at("architecture")).find("llama") != std::string::npos` inside building_blocks.cpp which turns off `reorder_interleaved_format`

Reply:
Got it, thanks, then the non-llama case should be handled.
Maybe we can consider to find a better name instead of create_llama_model to avoid confusion in next PR.